source/assets/sass/_breakpoints.scss

```diff
@@ -1,3 +1,5 @@
+@use "functions";
```

Comment:
the migrator uses the double quote so I went with it.

Reply:
Our formatter (Prettier) is currently set to single quotes, so `yarn lint` will change this.